pepper-apis/src/main/antlr4/org/broadinstitute/ddp/pex/lang/Pex.g4

```diff
@@ -20,14 +20,14 @@ expr
   ;
 
 query
-  : 'user' '.' study '.' studyPredicate                                                  # StudyQuery
-  | 'user' '.' study '.' form '.' formPredicate                                          # FormQuery
-  | 'user' '.' study '.' form '.' instance '.' formInstancePredicate                     # FormInstanceQuery
-  | 'user' '.' study '.' form '.' question '.' questionPredicate                         # QuestionQuery
-  | 'user' '.' study '.' form '.' question '.' 'answers' '.' predicate                   # DefaultLatestAnswerQuery
-  | 'user' '.' study '.' form '.' instance '.' question '.' 'answers' '.' predicate      # AnswerQuery
-  | 'user' '.' 'profile' '.' profileDataQuery   # ProfileQuery
-  | 'user' '.' 'event' '.' 'testResult' '.' testResultQuery   # EventTestResultQuery
+  : USER_TYPE '.' study '.' studyPredicate                                                  # StudyQuery
+  | USER_TYPE '.' study '.' form '.' formPredicate                                          # FormQuery
+  | USER_TYPE '.' study '.' form '.' instance '.' formInstancePredicate                     # FormInstanceQuery
+  | USER_TYPE '.' study '.' form '.' question '.' questionPredicate                         # QuestionQuery
+  | USER_TYPE '.' study '.' form '.' question '.' 'answers' '.' predicate                   # DefaultLatestAnswerQuery
+  | USER_TYPE '.' study '.' form '.' instance '.' question '.' 'answers' '.' predicate      # AnswerQuery
+  | USER_TYPE '.' 'profile' '.' profileDataQuery   # ProfileQuery
+  | USER_TYPE '.' 'event' '.' 'testResult' '.' testResultQuery   # EventTestResultQuery
```

Comment:
It looks like we only need to add operator support for the question queries, so adding operator support to all the other queries seems way too much. Also, for something like the `EventTestResultQuery` I don't think it makes sense to add operator support to that because currently events are for participants not operators.

How about we keep `EventTestResultQuery` and `StudyQuery` without operator support?

Reply:
I agree, that's might be too much. Removed operator from `EventTestResultQuery` and `StudyQuery`